index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
```

Comment:
- [ ]  Kindly maintain the project's Figma design in terms of colors, fonts, etc. Your form heading currently does not meet all the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Kindly fix this.

**This is how your project currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177409413-f5a04afd-a29d-4975-82f8-654987ac9881.png)



**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177409336-f9bdf6ec-ac8b-4f14-87dc-36dc24fb7a1b.png)

**Using `text-align: center;` should fix this**

Reply:
which buttons should be styled in my portfolio @omarsalem7